File: android/Pets-Starter/app/src/main/java/com/pets/viewModels/favorites/FavoritesViewModel.kt
package com.pets.viewModels.favorites

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.Observer
import com.pets.architecture.stores.Store
import com.pets.architecture.viewModels.ViewModel
import com.pets.domain.favorites.FavoritesEvent
import com.pets.domain.favorites.FavoritesReducer
import com.pets.domain.favorites.FavoritesState
import com.pets.domain.favorites.FetchList
import com.pets.models.Pet

File: ios/Pets-Starter/ViewModels/Sources/ViewModels/Favorites/FavoritesViewModel.swift
//
//  File.swift
//  
//
//  Created by Dylan Velez on 11/08/21.
//

import Foundation
import Architecture
import Domain
import Combine
import Models

public final class FavoritesViewModel: ViewModel {
    public typealias Event = FavoritesEvent
    public typealias FavoriteStore = Store<FavoritesState, FavoritesEvent,  FavoritesReducer>
}
